extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +127,75 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+ const renderOutlinkPageAttachmentUI = (win, pageEl, attachmentEl, attachmentHref) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+  
+  openAttachmentEl.addEventListener('click', () =>
+    openAttachmentEl.openAttachment()
+  );
+
+  // Setting theme  
+  const theme = attachmentEl.getAttribute('theme');
+  if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
+    openAttachmentEl.setAttribute('theme', AttachmentTheme.DARK);
+  }
```

Comment:
Can be replaced with `openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'))` since validation forces it to be lowercase